1-exercises/C-array-get-set/exercise.js

```diff
@@ -5,11 +5,11 @@
 */
 
 function first(arr) {
-  return; // complete this statement
+  return arr[0]; // complete this statement
 }
 
 function last(arr) {
-  return; // complete this statement
+  return arr.slice(-1); // complete this statement
```

Comment:
you could also do `arr[arr.length-1]` here which might be slightly faster, although I don't think it matters too much